components/ButtonNextPage/styles.js

```diff
@@ -0,0 +1,19 @@
+import styled from "styled-components";
+
+export const StyledButtonNextPage = styled.button`
+  padding: 15px 40px;
+  border: none;
+  border-radius: 100px;
+  text-decoration: none;
+  height: 70px;
+  width: 250px;
+  font-size: 20px;
+  background-color: #bcbdbc;
+`;
+
+export const StyledContainer = styled.div`
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  margin-left: 30px;
+`;
```

Comment:
You could think about using relative values ([Responsive Units](https://github.com/neuefische/bo-web-23-3/blob/main/sessions/css-responsive/css-responsive.md)) instead of absolute values. 
That would probably mean a major rebuild throughout your app, for that reason I suggest a refactor story for it that you wrote at some point in between.

Reply:
Yes, you're right. This is just basic styling for now. I will address it in a later user story